src/Apps/Order/Components/ArtworkSummaryItem.tsx

```diff
@@ -65,24 +68,27 @@ const ArtworkSummaryItem: React.FC<ArtworkSummaryItemProps> = ({
             display="inline"
           >
             {title}
-          </Text>
-          <Text variant="sm" color="black60" display="inline">
-            {date && `, ${date}`}
+            {!privateSale && date && `, ${date}`}
           </Text>
         </Box>
-        <Text variant="sm" overflowEllipsis color="black60">
-          {name}
-        </Text>
-        <Text variant="sm" color="black60">
-          {shippingOrigin}
-        </Text>
-        <Text variant="sm">
-          {artworkPrice &&
-            `${priceLabel} ${appendCurrencySymbol(
+        {!privateSale && (
+          <Text variant="sm" overflowEllipsis color="black60">
+            {name}
+          </Text>
+        )}
+        {!privateSale && (
+          <Text variant="sm" color="black60">
+            {shippingOrigin}
+          </Text>
+        )}
```

Comment:
suggestion: it could reduce the amount of checks if we could render `name` and `shippingOrigin` within one `!privateSale `check.

Reply:
good idea!